Dockerfile.prod

```diff
@@ -0,0 +1,9 @@
+FROM nginx:1.14
+
+EXPOSE 80 9000
+
+RUN rm /etc/nginx/conf.d/*
```

Comment:
this shouldn't be necessary. We only want to overwrite the default.conf file in the /etc/nginx/conf.d, which is what the next line is doing.

Reply:
Corrected this one as well...